weblate/checks/consistency.py

```diff
@@ -95,9 +95,14 @@ class TranslatedCheck(TargetCheck):
 
     check_id = "translated"
     name = _("Has been translated")
-    description = _("This string has been translated in the past")
+    description = ""
```

Comment:
Please keep the description here as well, it is used for example when listing all the checks (/checks/ URL).